_data/content/projects/es.yml

```diff
@@ -12,64 +12,192 @@ page_data:
-        # ‘Nuevas expografías: entre la tecnología y las experiencias’ seminario en  Coloquios Cultura Visual Contemporaneo series- Fundación Mainel, Valencia, España
+        Elisa fue invitada por la Fundación Mainel a presentar y participar
+        en un panel de líderes del sector que discutieron el papel de las
+        tecnologías y experiencias en la realización de exposiciones. Se
+        centró en varias tipologías en las que ha trabajado, desde las
+        exitosas exposiciones inmersivas del Museo V&A de Londres
+        (David Bowie Is, 'You Say You Want a Revolution?' y Pink Floyd:
+        Their Mortal Remains), Exposiciones Mundiales (particularmente
+        la Dubai Expo 2020) y experiencias de base tecnológica. El ciclo
+        de seminarios de Cultura Visual Contemporánea forma parte de
+        la oferta pública de la Mainel que se centra en la mejora de la
+        sociedad a través de la solidaridad y la cultura como
+        herramientas transformadoras, para conseguir que las personas
+        sean protagonistas de su propio desarrollo. En su presentación,
+        Elisa mostró cómo las experiencias inmersivas son clave para la
+        democratización de la cultura, ya que convierten a los visitantes
+        pasivos en participantes activos, cuya comprensión y
+        sentimientos son tan importantes como el conocimiento que
+        comparten los curadores y las instituciones mismos. Tanto los
+        temas de esta discusión como las motivaciones detrás de la
+        programación de Mainel muestran un profundo cuidado por el
+        público, cuyas vidas buscan enriquecer y empoderar. Ver
+        <a href="https://mainel.org/noticias/expografias-coloquio-cvc-27/" target="blank"> Nuevas expografías: se puede interactuar con la obra incluso sin que esté físicamente presente · Fundación Mainel
 
     - type: id_archives
-      project_name: "Rise-Rosa-Rage Archivo de Gráficos Sociopolíticos"
-      project_excerpt: ""
-      project_ubication: "Roma, Italia / Online"
+      project_name: "Archivo Rise-Rosa-Rage de gráfica sociopolítica"
+      project_excerpt: "Curaduría, Gestión y Desarrollo de Colecciones, Catalogación"
+      project_ubication: "Roma, Italia / En línea"
       img: ":Rise_Rosa_Rage.webp"
       date: "En curso desde 2016"
       post: |
-        # Rise-Rosa-Rage Archivo de Gráficos Sociopolíticos, Roma, Italia / Online
+        Rise-Rosa-Rage es un archivo de gráfica sociopolítica que
+        actualmente consta de más de 4.000 objetos físicos, digitales y en
+        la biblioteca de referencia. CuraCultura es responsable de la
+        curación, el desarrollo de colecciones y la gestión del archivo, así
+        como de la catalogación en línea, la traducción de contenidos, la
+        redacción de las cartelas de los objetos y de artículos, y el
+        desarrollo web. Las colecciones datan predominantemente de las
+        décadas de los 1960 hasta los 1990, con algunas piezas también
```

Comment:
In English translation we have 1960 to 1980, and here in Spanish we have it from 1960 to 1990. maybe this should be asked to the client